packages/dds/tree/src/test/util/summaryBenchmarkSizes.ts

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { IsoBuffer } from "@fluidframework/common-utils";
+import { TransactionResult } from "../../checkout";
+import { singleTextCursor } from "../../feature-libraries";
+import { ISharedTree } from "../../shared-tree";
+import { detachedFieldAsKey, TreeValue } from "../../tree";
+import { brand } from "../../util";
+import { TestTreeProvider } from "../utils";
+
+const random = makeRandom(129384);
+
+export interface OutputType {
+    noEdits: number,
+    insert1: number,
+    insert10: number,
+    insert100: number,
+    insertRemove1: number,
+    insertRemove10: number,
+    insertRemove100: number,
+}
+
+export async function getSummaryBenchmarkSizes(): Promise<OutputType> {
```

Comment:
doc comments on exported things would be good.

Reply:
Added doc comments